src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -160,13 +161,14 @@ const validFormConfigKeys = formConfig => {
   });
 };
 
-const validFormId = formConfig => {
+const validVetsJsonSchema = formConfig => {
+  if (formConfig.hasVetsJsonSchema === false) {
```

Comment:
My tip would be to invert this property and make it something like `noVetsJsonSchema` or `hasNoVetsJsonSchema` or `missingFromVetsJsonSchema` (like it is in this file) or similar. That way you don't have to explicitly test for `false` and can just test for a truthy value. Because the only reason this works right now is due to all the already existing forms having `formConfig.hasVetsJsonSchema` as `undefined`, so to me that's kind of confusing because they have falsey values, but not specifically `false`. I feel like it works better to have a property that you need to explicitly set as `true` and then all the old forms can have their falsey values.

Reply:
good callout with the true/false inversion. mottie gave me some similar feedback. for now I updated it to `skipVetsJsonSchemaCheck`, but we can still adjust if we want.